packages/kit/src/types/public.d.ts

```diff
@@ -1053,8 +1052,8 @@ export interface RouteDefinition<Config = any> {
 	config: Config;
 }
 
-export class Server {
-	constructor(manifest: SSRManifest);
+export interface Server {
+	new (manifest: SSRManifest): Server;
```

Comment:
Why this change? Curious if these are equivalent.

Reply:
ah this is because at one point i was experimenting with having `.ts` files in `src` instead of `.d.ts` files, so that all the `.d.ts` files would be generated. (i think having `.d.ts` files in `src` is better, on reflection — we just need to copy them over to the `types` directory). you can't declare a `class` in a `.ts` file without actually implementing it

now that it's in a `.d.ts` again we could revert